types/shares_test.go

```diff
@@ -285,7 +287,7 @@ func TestDataFromSquare(t *testing.T) {
 func TestFuzz_DataFromSquare(t *testing.T) {
 	t.Skip()
 	// run random shares through processContiguousShares for a minute
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*20)
```

Comment:
Why did we reduce this?

Reply:
not sure, but I likely accidently switched this when trying to help Tomasz debug and forgot to change it back. Changed it back here. https://github.com/celestiaorg/celestia-core/pull/637/commits/7d2505d9c2862ecbac45c4e8c93dab9de4ac7380